src/libs/getPlaidDesktopMessage/index.ts

```diff
@@ -0,0 +1,5 @@
+import GetPlaidDesktopMessage from './types';
+
+const getPlaidDesktopMessage: GetPlaidDesktopMessage = () => {};
```

Comment:
```suggestion
const getPlaidDesktopMessage: GetPlaidDesktopMessage = () => undefined;
```

This is very odd, looking at the place where it is used, [we check if value is truthy](https://github.com/Expensify/App/blob/main/src/pages/ReimbursementAccount/BankAccountStep.js#L112), and for a function like that the return value is `undefined`. So I will suggest to return `undefined` instead of `{}`. Wdyt?